app/jobs/stream_big_query_participant_declarations_job.rb

```diff
@@ -1,10 +1,6 @@
 # frozen_string_literal: true
 
-class StreamBigQueryParticipantDeclarationsJob < CronJob
-  self.cron_expression = "0 * * * *"
-
-  queue_as :big_query_participant_declarations
-
+class StreamBigQueryParticipantDeclarationsJob < ApplicationJob
```

Comment:
I think this might fail in non-production environments?

Reply:
From what I remember, bigquery will give you a 200 response saying something like "didn't work" if there is a problem 😄. So it is probably "failing" like this without erroring in all environments right now, but yes it will benefit from some short circuiting based on the env